zds/tutorialv2/forms.py

```diff
@@ -635,7 +636,13 @@ class AcceptValidationForm(forms.Form):
         required=False,
         initial=True
     )
-
+    pubdate = forms.DateField(
+        label=_(u'Date de publication (Laissez vide pour maintenant.)'),
```

Comment:
Je mettrai "pour publier immédiatement" quand je repasserai sur cette PR.

2016-07-15 20:13 GMT+02:00 Laville Augustin <notifications@github.com>:

> In zds/tutorialv2/forms.py
> <https://github.com/zestedesavoir/zds-site/pull/3276#discussion_r71017337>
> :
>
> > @@ -635,7 +636,13 @@ class AcceptValidationForm(forms.Form):
> >          required=False,
> >          initial=True
> >      )
> > -
> > +    pubdate = forms.DateField(
> > +        label=_(u'Date de publication (Laissez vide pour maintenant.)'),
>
> +1 j'avais pas compris sans regarder le contexte et l'issue
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zestedesavoir/zds-site/pull/3276/files/b5e0edf701ffee84f83c42cdb8bb8673cecd8e9e#r71017337>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABc_xRMkXFsGBT5my3S13Zmgu4tRf2Z3ks5qV83igaJpZM4G_EBV>
> .
>